frontend/test/pytest/test_transform.py

```diff
@@ -365,23 +366,20 @@ def qfunc():
         _, observed_shape = jax.tree_util.tree_flatten(observed)
         assert expected_shape == observed_shape
 
-
-class TestSumExpand:
-    """Test Sum Expand"""
-
-    def test_sum_expand(self, backend):
-        """Test Sum Expand"""
+    def test_split_non_commuting_mulitiple_observables(self, backend):
+        """Test split_non_commuting on two separate measurements with non-commuting
+        observables"""
 
         def qnode_builder(device_name):
             """Builder"""
 
-            @sum_expand
+            @qml.transforms.split_non_commuting
             @qml.qnode(qml.device(device_name, wires=2, shots=None))
             def qfunc():
                 """Example taken from PL tests"""
                 obs1 = qml.prod(qml.PauliX(0), qml.PauliX(1))
                 obs2 = qml.prod(qml.PauliX(0), qml.PauliY(1))
-                return [qml.expval(obs1), qml.expval(obs2)]
+                return qml.expval(obs1), qml.expval(obs2)
```

Comment:
I would say cosmetic? 

With the list, the transform runs fine, and the returned values still matched expectation, but `assert expected_shape == observed_shape` failed because the `observed_shape` returned a list, and the `expected_shape` a tuple, so the PyTrees weren't the same.